fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/DataEncryptionService.java

```diff
@@ -0,0 +1,5 @@
+package de.adorsys.opba.fintech.impl.service;
+
+public interface DataEncryptionService {
+    String encrypt(String value);
```

Comment:
It is not `encrypt` it is `sign` - we sign the request. It should be RequestSigningService

Reply:
Renamed